logdna/index.js

```diff
@@ -1,11 +1,12 @@
+// Internal Libraries
+const cloudWatch = require('./lib/cloudWatch.js').createLogs;
+const pkg = require('./package.json');
+
 // External Libraries
 const agent = require('agentkeepalive');
 const asyncRetry = require('async').retry;
 const request = require('request');
-const zlib = require('zlib');
 
-// Constants
-const MAX_LINE_LENGTH = parseInt(process.env.LOGDNA_MAX_LINE_LENGTH) || 32000;
 const MAX_REQUEST_TIMEOUT_MS = parseInt(process.env.LOGDNA_MAX_REQUEST_TIMEOUT) || 30000;
```

Comment:
In that way, these constants could be part of that class

Reply:
What do you mean?

Reply:
We can have these constants inside the constructor of the `class CloudWatch` if we decide to build in that way